src/mlpack/methods/ann/layer/repeat_impl.hpp

```diff
@@ -0,0 +1,206 @@
+/**
+ * @file methods/ann/layer/repeat_impl.hpp
+ * @author Adam Kropp
+ *
+ * Implementation of the Repeat class, which repeats the input n times
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_REPEAT_IMPL_HPP
+#define MLPACK_METHODS_ANN_LAYER_REPEAT_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include <utility>
```

Comment:
Is this needed?  If so (I'm not sure what you're including it for), we could put it in `core.hpp` or `prereqs.hpp` if it would be useful in other parts of the library too.